src/test/scala/org/broadinstitute/dsde/agora/server/AgoraTestData.scala

```diff
@@ -639,22 +689,4 @@ object AgoraTestData {
     payload = payloadWithInvalidPersonalDockerTagNameInWdl,
     entityType = Option(AgoraEntityType.Task)
   )
-
-  def populateDatabase(agoraBusiness: AgoraBusiness) = {
```

Comment:
I think it was left over from before we implemented the beforeAll / afterAll db management stuff. Happy to see it killed